src/cluster/index.rst

```diff
@@ -17,8 +17,8 @@ Cluster Management
 ==================
 
 As of CouchDB 2.0.0, CouchDB can be run in two different modes of operation:
-    * Standalone
-    * Cluster
+    * Standalone: The default mode of operation. Each instance is a single logical entity. Typically replication is setup to other standalone CouchDB instances.
```

Comment:
Standalone isn't the "default" mode of operation. It may be your preference, but it's not the global expectation, nor is it our recommendation (that would be a cluster of 3 nodes for production use).

Reply:
The second sentence is also inaccurate here. Something like this is more accurate:

"In this mode, CouchDB's clustering is unavailable. CouchDB's HTML-based replication with other CouchDB installations remains available."

Reply:
I swapped HTML for HTTP, I guess that was your intention